Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/FixedFragmentStatePagerAdapter.kt

```diff
@@ -57,7 +57,7 @@ abstract class FixedFragmentStatePagerAdapter(fm: FragmentManager) : FragmentSta
      * @param object the Fragment we want to check for
      * @return the position if found else -1
      */
-    protected fun findFragmentPositionHashMap(`object`: Fragment): Int {
+    private fun findFragmentPositionHashMap(`object`: Fragment): Int {
```

Comment:
would be easier to just use another variable name than backquote Kotlin keywords ;-)

Reply:
😅